app/services/analytics.rb

```diff
@@ -26,6 +27,7 @@ def track_event(event, attributes = {})
       locale: I18n.locale,
     }
 
+    analytics_hash.merge!(irs_session_id: irs_session_id) if irs_session_id
```

Comment:
I feel like we could do this in one step? but it's not a huge deal either way
```suggestion
      locale: I18n.locale,
      irs_session_id: irs_session_id,
    }.compact
```

Reply:
Could that have side-effects if any of the other parameters were nil? Or does that never happen?